tests/test_parser.py

```diff
@@ -127,7 +127,7 @@
     ("""<track  src=bla>""", """<track src="bla"/>"""),
     ("""<video  src=bla>""", """<video src="bla"></video>"""),
     # Test inserted tag s
-    ("""<b><a></a></b>""", """<b><a></a></b>"""),
+    ("""<b><a></a></b>""", """<b></b><a></a>"""),
```

Comment:
all these malformed html correction changes look like improvements/bugfixes to me!

Reply:
So far, so good. The next PR will show a difference.